src/utils/useFeatureFlag.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { ServiceConfig } from "../api/ServiceContext";
+import useServiceConfig from "../api/useServiceConfig";
+
+/**
+ * Get the current state of the target feature flag.
+ * @See feature flags in Service Config JSON.
+ * @param feature: string name of the target flag.
+ * @returns enabled: boolean true, the feature has been enabled.
+ *    False, the target feature is disabled or the flag was not found.
+ */
+export default function useFeature(feature: string): boolean {
```

Comment:
should we keep this in individual projects or should we have a hook in madie-util that all apps can use?

Reply:
I was thinking it was an "either or" situation. 

Moving it into util would require duplicating the logic for reading the service config that's already in each project that consumes an api. 

Having it each project that needs feature flag support duplicates the function proposed in this PR.

Either way uses the same source of truth so there shouldn't be a risk of App 1 and App 2 reading different results for a feature flag.

Reply:
Yeah for how it works right now that's true. I think we should consider raising a tech debt item to change how we load service config in the future. We could move that fetch code to madie-util as well, and cache the value there so it's only retrieved from the backend/S3 once.